pkg/opensearch/snapshot_tests/testdata/lucene_logs.query_input.json

```diff
@@ -7,14 +7,13 @@
     "alias": "",
     "bucketAggs": [
       {
-        "field": "timestamp",
+        "field": "agent.keyword",
         "id": "2",
         "settings": {
-          "interval": "20m",
-          "min_doc_count": "0",
-          "trimEdges": "0"
+          "interval": "1000",
+          "min_doc_count": "15"
         },
-        "type": "date_histogram"
+        "type": "histogram"
```

Comment:
Test now shows that even with "histogram" and "min_doc_count" set, query is built with default date_histogram